src/System.Security.Cryptography.Primitives/tests/HmacAlgorithmTest.cs

```diff
@@ -67,6 +71,8 @@ public void ReassignAlgorithmName()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework,
+            "Change needs to be ported to netfx")]
```

Comment:
What's the difference? Why? Should we be changing corefx instead of asserting that we're planning on changing netfx?